src/Components/ResourceList/Card/ResourceCardName.scss

```diff
@@ -13,10 +13,12 @@
 }
 
 .cf-resource-name--text {
+  display: flex;
+  align-items: center;
+
   font-size: $cf-card--name-font-size;
   font-weight: $cf-card--name-font-weight;
   font-family: $cf-text-font;
-  display: inline-block;
```

Comment:
this selector is also used in more places than dashboards and is something you'll want to take great care changing. Changing from `inline-block` to `flex` might have very odd changes in layout. It might be better to try `inline-flex`, but even then, I'd be very careful changing that, as it might have some strange, unintended consequences beyond just the cards you tested them in.